### PR TITLE
Switch the E1.31 tests over to using StdinHandler and fix some minor …

### PR DESCRIPTION
**Pull request by peternewman**
…bugs

### CODE REVIEW

**Comment by peternewman** [comment]
This should allow us to enable the E1.31 code on Windows, once we've made StdinHandler work properly on Windows.

File: plugins/e131/e131/E131TestFramework.h [hunk 1]
```diff
@@ -320,8 +320,8 @@ class StateManager {
     ola::acn::CID m_cid1, m_cid2;
     ola::plugin::e131::E131Node *m_local_node, *m_node1, *m_node2;
     ola::io::SelectServer *m_ss;
+    ola::io::StdinHandler *m_stdin_handler;
```
Comment (nomis52) [inline plugins/e131/e131/E131TestFramework.h]: use auto_ptr

Comment (peternewman) [inline plugins/e131/e131/E131TestFramework.h]: Just for the StdinHandler?

Comment (peternewman) [inline plugins/e131/e131/E131TestFramework.h]: Done (for StdinHandler)

**Comment by nomis52** [comment]
LGTM

**Pull request merged**